Processes/Docs-Contribution-Process-Internal.md

```diff
@@ -0,0 +1,105 @@
+
+As a content contributor you are also responsible to review the final submission before it is published. Your review is reflective of the accuracy and completeness of the content.
+
+### When to Use this Procedure
+Use this procedure when you need to create or update content from one or repositories to  [EOS Documentation](https://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption in EOS Documentation.
+
+### Stages of Publication with Contributor Roles Identified
+1. Author: Create branch
+2. Author: Create and name document
+3. Author: Apply template and write content
+4. Author: Submit to DevRel
+5. DevRel: Refine submission according to technical writing best practices
+6. Author: Add changes, if necessary
+7. Author: Final review and approval
+
+
+## Write Content for the EOS Documentation Portal
+Provide content to the DevRel team in Markdown (.md) format.
+
+The ENF Developer Relations team (DevRel) developed a set of templates to assist development of documentation for the EOS ecosystem. The templates are written in Markdown (.md) format and provide a structure for your content contribution. Take advantage of these writing tools to assist with your content contribution. These templates are guidelines for various types of content and will help you organize your content with a common look and feel. These templates are not a substitute for content accuracy and completeness.
+Complete the following steps to apply a template to your content:
+1. Open a new document in your GitHub branch web editor
+2. Open the DevRel repository **Template** Folder
+3. Open the template file for your document type. 
+
+      a. Overview  
+      b. Explainer  
+      c. Tutorial.  
+      d. How To
+4. Copy the contents of the template file
+5. Paste the contents of the template file into your new document in your GitHub branch web editor
+
+   The template provides a structure for your content
+6. Write your content
+
+   Use the topic designations provided by the template to organize your content  
+
+7. Submit the document containing your content to DevRel
+
+## Submit Content for Inclusion in EOS Documentation Portal
+
+**Scenario**: You have written content that needs to be added to the EOS Documentation portal It is in .md format. You wrote the content using HackMD, or the GitHub Web Editor, or in Google Docs and converted it to .md format. Now what do you need to do?
+
+Complete the following steps to submit your content to DevRel:
+
+1. Fill out the submission form with:
+* Your Name
+* Title of document
+* Document type
+  * Overview
+  * Explainer
+  * Tutorial
+  * How To
+* What role the document pertains to
```

Comment:
I think this will be obvious in the future when we restructure the directories since there will be clear directories for each

Reply:
The author/SME may know a role that the feature is intended for, the audience for the new feature. This information is helpful and adds knowledge for the DevRel team.

Reply:
I'll change when necessary after the restructure.

Done